s2n.mk

```diff
@@ -114,7 +114,7 @@ endif
 
 ifdef S2N_GDB
     S2N_DEBUG = 1
-    CFLAGS += -Og
+    CFLAGS += -O0
```

Comment:
Why are we changing this flag to `-O0` in this PR?

Reply:
Lindsay apparently was noticing that -Og was skipping around a fair bit still, so I added this change. It's not actually relevant to the PR :)

Reply:
I would not include this change in this PR